### PR TITLE
191 add app for susbtrates layers and growth run to movpe2 in ikz plugin

### PR DESCRIPTION
**Pull request by budschi**
I added a first version of a Substrate app for MOVPE. It will provide a table with the most important quantities according to Ta-Shuns data revision. Additional search widgets can be added later.
grafik

### CODE REVIEW

File: IKZ_plugin/pyproject.toml [hunk 1]
```diff
@@ -132,5 +132,6 @@ dir_sol_schema = "ikz_plugin.directional_solidification:schema"
 dir_sol_manual_protocol_excel_parser = "ikz_plugin.directional_solidification:excel_parser"
 czochralski_schema = "ikz_plugin.czochralski:schema"
 czochralski_multilog_parser = "ikz_plugin.czochralski:parser"
+movpe_app = "ikz_plugin.movpe.movpe_app:movpesubstrateapp"
```
Comment (aalbino2) [inline IKZ_plugin/pyproject.toml:135]:
```suggestion
substrate_app = "ikz_plugin.movpe.movpe_app:substrate_app"
```

File: IKZ_plugin/src/ikz_plugin/movpe/movpe_app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+from nomad.config.models.ui import (
+    App,
+    Column,
+    Columns,
+    FilterMenu,
+    FilterMenus,
+    Filters,
+)
+
+movpesubstrateapp = AppEntryPoint(
```
Comment (aalbino2) [inline IKZ_plugin/src/ikz_plugin/movpe/movpe_app/__init__.py:12]:
```suggestion
substrate_app = AppEntryPoint(
```

**Review by aalbino2 — CHANGES_REQUESTED** [review]
How about changing the app name to make is general and not only movpe-oriented?

File: IKZ_plugin/pyproject.toml (hunk 1 above)
Comment (budschi) [reply IKZ_plugin/pyproject.toml:135]: in principle yes. But I am only allowing entries of type `SubstrateMovpe`

Comment (aalbino2) [reply IKZ_plugin/pyproject.toml:135]: I see. okay, we will maybe change it in the future when we manage to have a single substrate class for IKZ

**Review by aalbino2 — APPROVED** [review]

**Pull request merged**